synchronizer/ext_control.go

```diff
@@ -0,0 +1,112 @@
+package synchronizer
```

Comment:
Can you add somewhere a readme with an example of the file content to send commands?